Task1/todo.js

```diff
@@ -0,0 +1,121 @@
+// This is the array that will hold the todo list items
+let todoItems = [];
+// This function will create a new todo object with text input, 
+//check to see which task is completed and id for remove
+
+
+// Render the todo items
+function renderTodo(todo)
+{
+    const list=document.querySelector("#todoList");
+    const item=document.querySelector(`[todo-id='${todo.id}']`);
+
+    if(todo.deleted)
+    {
+        item.remove();
+        return
+    }
+    const node=document.createElement("li");
+    node.setAttribute('todo-id',todo.id);
+    const deleteButton =document.createElement("button");
+    deleteButton.classList.add('js-delete-todo');
+    deleteButton.innerHTML="Delete" 
+    node.innerHTML = `
+    <span>${todo.text}</span>
+    `;
+    node.appendChild(deleteButton)
+    list.appendChild(node)
+}
+function addTodo(text)
+{
+    const todo={
+        text,
+        id:Date.now(),
+    };
+
+    todoItems.push(todo);
+    renderTodo(todo);
+}
+
+function deleteTodo(id)
+{
+    const index=todoItems.findIndex(item=>item.id===Number(id));
```

Comment:
This can be used in a destructured from as well:

`const index=todoItems.findIndex({id}=>id === Number(id))`

Also, try to use spacing in between i.e. indentation. This helps to maintain the readability of the code.

@Poorvi-3pillar

Reply:
okay